term-manager/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<name>sakai-acadtermmanage - Base</name>
+	<groupId>org.sakaiproject.acadtermmanage</groupId>
+	<artifactId>sakai-acadtermmanage</artifactId>	
+	<packaging>pom</packaging>
+	
+	<parent>
+		<groupId>org.sakaiproject</groupId>
+		<artifactId>master</artifactId>
+		<version>20-SNAPSHOT</version>
+		<relativePath />
+	</parent>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<my.wicket.version>6.29.0</my.wicket.version>
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+
+			<!-- project dependencies -->
+			<dependency>
+				<groupId>org.sakaiproject.acadtermmanage</groupId>
+				<artifactId>sakai-acadtermmanage-api</artifactId>
+				<version>${project.version}</version>
+				<scope>provided</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.sakaiproject.acadtermmanage</groupId>
+				<artifactId>sakai-acadtermmanage-impl</artifactId>
+				<version>${project.version}</version>
+				<scope>runtime</scope>
+			</dependency>
+
+
+			<!-- third party dependencies -->
+			<dependency>
```

Comment:
wicket dependencies should only exist in the tool so no need to declare here

Reply:
This is just in dependency management the same way other tools are defining it, so going to leave it here.